src/System.Runtime.InteropServices.RuntimeInformation/tests/DescriptionNameTests.cs

```diff
@@ -27,7 +27,7 @@ public void DumpRuntimeInformationToConsole()
 
             Console.WriteLine($@"{dvs} OS={osd} OSVer={osv} OSArch={osa} Arch={pra} Framework={frd} LibcRelease={lcr} LibcVersion={lcv}");
 
-            string tempDriveFormat = new DriveInfo(GetTestFilePath()).DriveFormat;
+            string tempDriveFormat = new DriveInfo(Path.GetTempPath()).DriveFormat;
```

Comment:
This should fix the tempDriveFormat case (you can undo the `: FileCleanupTestBase` change).

But what about the next line?  Won't that still fail?

Reply:
Why - the location exists?

I guess I will find out.

Reply:
It works:
```
2018-05-04 13:57:10,854: INFO: proc(54): run_and_log_output: Output: Distro=fedora VersionId=26.0 OS=Linux 4.14.15-200.fc26.x86_64 #1 SMP Wed Jan 24 04:26:15 UTC 2018 OSVer=Unix 4.14.15.200 OSArch=X64 Arch=X64 Framework=.NET Core 4.6.26504.05 LibcRelease=stable LibcVersion=2.25
2018-05-04 13:57:10,866: INFO: proc(54): run_and_log_output: Output: temp on tmpfs, binaries on xfs
```